frontend/src/main/components/SectionsTableBase.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { useTable, _useSortBy, useGroupBy, useExpanded } from 'react-table'
```

Comment:
If we're not using `useSortBy`, can we remove it from the import list instead of adding the underscore?

```suggestion
import { useTable, useGroupBy, useExpanded } from 'react-table'
```